pkgs/tools/X11/opentabletdriver/default.nix

```diff
@@ -87,8 +89,8 @@ buildDotnetModule rec {
     install -Dm644 $src/OpenTabletDriver.UX/Assets/otd.png -t $out/share/pixmaps
 
     # TODO: Ideally this should be build from OpenTabletDriver/OpenTabletDriver-udev instead
-    dpkg-deb --fsys-tarfile ${debPkg} | tar xf - ./usr/lib/udev/rules.d/99-opentabletdriver.rules
-    install -Dm644 ./usr/lib/udev/rules.d/99-opentabletdriver.rules -t $out/lib/udev/rules.d
+    dpkg-deb --fsys-tarfile ${debPkg} | tar xf - ./usr/lib/udev/rules.d/70-opentabletdriver.rules
+    install -Dm644 ./usr/lib/udev/rules.d/70-opentabletdriver.rules -t $out/lib/udev/rules.d
```

Comment:
Can we use this script instead of copying the rules from the `.deb` package?
https://github.com/OpenTabletDriver/OpenTabletDriver/blob/master/generate-rules.sh.

This way we can simplify the build and drop the download from the `.deb`.

Reply:
Updating udev rules broke the NixOS test
This line is causing some trouble: `KERNEL=="js[0-9]*", SUBSYSTEM=="input", ATTRS{name}=="OpenTabletDriver Virtual Tablet", RUN+="/usr/bin/env rm %E{DEVNAME}"`
And the test fails with this message: `/usr/bin/env is called in udev rules but is not executable or does not exist`

Using generate-rules.sh script instead of the deb package to obtain udev rules works, however it could be improved:
- the script depends on git and bash, pulling in unnecessary dependencies, should we patch these out from the script if possible?
- running the script with `./generate-rules.sh` results in the error `./generate-rules.sh: cannot execute: required file not found`, as a workaround I'm running it with `bash generate-rules.sh`

I'll get back to this PR in a few days, in the meantime any help is well appreciated :)

Reply:
> KERNEL=="js[0-9]*", SUBSYSTEM=="input", ATTRS{name}=="OpenTabletDriver Virtual Tablet", RUN+="/usr/bin/env rm %E{DEVNAME}"

Replace the `/usr/bin/env rm` with `${lib.getExe' pkgs.coreutils "rm"}` and it should work.

> the script depends on git and bash, pulling in unnecessary dependencies, should we patch these out from the script if possible?

For `git`, it seems the only [reason it exists](https://github.com/OpenTabletDriver/OpenTabletDriver/blob/v0.6.4.0/generate-rules.sh#L26) is to get the root of the repository, which could be replaced with `$src`.

> running the script with ./generate-rules.sh results in the error ./generate-rules.sh: cannot execute: required file not found, as a workaround I'm running it with bash generate-rules.sh

You can call `patchShebangs generate-rules.sh` in `preBuild` to patch the shebang.

Reply:
Thanks for your help, it saved me hours!

NixOS test is passing again, the package no longer depends on git and the script can be called with `./`

I also removed the chmod, since the file doesn't need to be writable, and in the nix store it can't be